recipe/meta.yaml

```diff
@@ -41,6 +41,7 @@ outputs:
   - name: clang_win-64
     script: install-pkg.sh   # [not win]
     script: install-pkg.bat  # [win]
+    skip: True  # [not win]
```

Comment:
If we skip everything but `win` here, do we still need any of the `not win` lines and all the selectors in this output?

Reply:
yes, that is an conda-build issue ... it is adviced to specify a separete build-script, if a selector is used for it.